packages/react/src/components/List/List.tsx

```diff
@@ -140,10 +140,10 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
   private _scrollTop: number;
   private _pageCache: IPageCache<T>;
 
-  public static getDerivedStateFromProps<T = any>(
```

Comment:
There's already a `T` defined at the class level. Open to other naming suggestions.